### PR TITLE
manifest: Updated Matter repository to pull commissioning fix

### PR DESCRIPTION
**Pull request by kkasperczyk-no**
The Matter version was updated to pull a fix for commissioning window.

### CODE REVIEW

File: doc/nrf/releases_and_maturity/known_issues.rst [hunk 1]
```diff
@@ -380,6 +380,14 @@ Matter
 
 The issues in this section are related to the :ref:`ug_matter` protocol.
 
+.. rst-class:: v2-7-0
+
+KRKNWK-19443: The device commissioning window can be opened for longer than 900 seconds, what violates the Matter specification.
```
Comment (doublemis1) [inline doc/nrf/releases_and_maturity/known_issues.rst:385]: Can we add information that the issue impacts following Matter Certification Test Cases:
[TC-CADMIN-1.21] Open commissioning window - durations max and max+1 [BCM] [DUT - Commissionee]
[TC-CADMIN-1.22] Open commissioning window - durations max and max+1 [ECM] [DUT - Commissionee]

**Review by doublemis1 — APPROVED** [review]

File: doc/nrf/releases_and_maturity/known_issues.rst [hunk 2]
```diff
@@ -380,6 +380,15 @@ Matter
...
```
Comment (wiba-nordic) [inline doc/nrf/releases_and_maturity/known_issues.rst:385]:
```suggestion
KRKNWK-19443: The device commissioning window can be opened for longer than 900 seconds, which violates the Matter specification
```

File: doc/nrf/releases_and_maturity/known_issues.rst [hunk 3]
```diff
@@ -380,6 +380,15 @@ Matter
...
+  The device using Extended Announcement feature is allowed to open commissioning window for longer than 900 s only if it's uncommissioned.
```
Comment (wiba-nordic) [inline doc/nrf/releases_and_maturity/known_issues.rst:386]:
```suggestion
  A device using the Extended Announcement feature is allowed to open the commissioning window for longer than 900 s only if it is uncommissioned.
```

File: doc/nrf/releases_and_maturity/known_issues.rst [hunk 4]
```diff
@@ -380,6 +380,15 @@ Matter
...
+  However, the implementation with Extended Announcement enabled allows to open commissioning window for longer than 900 s if the device is commissioned as well.
```
Comment (wiba-nordic) [inline doc/nrf/releases_and_maturity/known_issues.rst:387]:
```suggestion
  However, the implementation with Extended Announcement enabled allows the device to open the commissioning window for longer than 900 s even when the device is commissioned.
```

File: doc/nrf/releases_and_maturity/known_issues.rst [hunk 5]
```diff
@@ -380,6 +380,15 @@ Matter
...
+  It leads to the TC-CADMIN-1.21 and TC-CADMIN-1.22 certification test cases failure.
```
Comment (wiba-nordic) [inline doc/nrf/releases_and_maturity/known_issues.rst:388]:
```suggestion
  This leads to the TC-CADMIN-1.21 and TC-CADMIN-1.22 certification test cases failure.
```

**Review by wiba-nordic — CHANGES_REQUESTED** [review]

**Review by wiba-nordic — APPROVED** [review]

**Review by ArekBalysNordic — APPROVED** [review]

**Pull request merged**